### PR TITLE
Create example_jsonlist.py

### PR DESCRIPTION
**Pull request by add30417**
The standard output of the script, is a list with the devices + status information in JSON Format.

### CODE REVIEW

File: examples/example_jsonlist.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    gateway = Gateway()
+
+    devices_command = gateway.get_devices()
+    devices_commands = await api(devices_command)
+    devices = await api(devices_commands)
+
+    list_devices = [dev for dev in devices]
+    #print("{\"DEVICES\":[", end= '');
+    print("[", end= '');
```
Comment (ggravlingen) [inline examples/example_jsonlist.py:92]: Any way we can turn this into a util function and take it as an argument in the existing example file?

**Review by ggravlingen — COMMENTED** [review]
Haven't seen anyone request this feature before. I'm curious, what is the use case?

File: examples/example_jsonlist.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import argparse
+import asyncio
+import uuid
+
+from pytradfri import Gateway
+from pytradfri.api.aiocoap_api import APIFactory
+from pytradfri.error import PytradfriError
+from pytradfri.util import load_json, save_json
+
+#CONFIG_FILE = "tradfri_standalone_psk.conf"
```
Comment (ggravlingen) [inline examples/example_jsonlist.py:32]: We should remove stale comments.

File: examples/example_jsonlist.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    api = api_factory.request
+
+    gateway = Gateway()
+
+    devices_command = gateway.get_devices()
+    devices_commands = await api(devices_command)
+    devices = await api(devices_commands)
+
+    list_devices = [dev for dev in devices]
+    #print("{\"DEVICES\":[", end= '');
```
Comment (ggravlingen) [inline examples/example_jsonlist.py:91]: We should remove stale comments.

File: examples/example_jsonlist.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+            print("\",\"TYPE\":\"SOCKET\"", end='');
+        elif device.has_light_control:
+            print(device.light_control.lights[0].dimmer, end='');
+            print("\",\"TYPE\":\"LIGHT\"", end='');
+        else:
+            print("\",\"TYPE\":\"UNKOWN\"", end='');
+        print(",\"TYPENAME\":\"" + device.device_info.model_number + "\",\"REACHABLE\":\"" + str(device.reachable) + "\"}", end='');
+        first=False;
+    print("]");
+    #print("]}");
```
Comment (ggravlingen) [inline examples/example_jsonlist.py:109]: We should remove stale comments.

**Comment by add30417** [comment]
I’m using this feature, to retrieve the current status for my smart home system.

File: examples/example_jsonlist.py (hunk 4 above)
Comment (add30417) [reply examples/example_jsonlist.py:109]: Stollen? It is kind of fork from other example. Why not use good comments?

Comment (ggravlingen) [reply examples/example_jsonlist.py:109]: Not stolen, comments that are commented out, ie # before them

Comment (add30417) [reply examples/example_jsonlist.py:109]: Sorry, well of course can be removed.

File: examples/example_jsonlist.py (hunk 1 above)
Comment (add30417) [reply examples/example_jsonlist.py:92]: Well, it is depending of design guideline. I would keep it separated. Also better for future change and extensions at both files.

**Pull request closed without merging**